src/query_raw.rs

```diff
@@ -0,0 +1,226 @@
+//! Raw SQL queries without parameter binding.
+//!
+//! [`QueryRaw`] executes SQL as-is without parameter binding support.
+//! Question marks are escaped using an odd/even strategy:
+//! - `?` → `??` (odd count, add one more)
+//! - `??` → `??` (even count, unchanged)
+use crate::{error::Result, row::Row, sql::Bind, Client};
+use serde::Serialize;
+
+pub use crate::cursors::{BytesCursor, RowCursor};
+use crate::query::Query;
+
+#[must_use]
+#[derive(Clone)]
+pub struct QueryRaw {
+    query: Query,
+}
+
+impl QueryRaw {
+    pub(crate) fn new(client: &Client, template: &str) -> Self {
+        // Raw queries apply odd/even question mark strategy:
+        // - Consecutive odd number of ? (1,3,5...) -> add one more ?
+        // - Consecutive even number of ? (2,4,6...) -> keep as is
+        Self {
+            query: Query::new_raw(client, template),
+        }
+    }
+
+    /// Display the SQL query with question mark escaping applied.
+    pub fn sql_display(&self) -> &impl std::fmt::Display {
+        self.query.sql_display()
+    }
+
+    /// Attempts to bind a value to the query.
+    ///
+    /// # Panics
+    /// Always panics - raw queries don't support parameter binding.
+    #[track_caller]
+    pub fn bind(self, _value: impl Bind) -> Self {
+        panic!("cannot bind parameters to raw query - use regular query() for parameter binding");
+    }
+
+    /// Attempts to set server-side parameters for the query.
+    ///
+    /// # Panics
+    /// Always panics - raw queries don't support parameter binding.
+    #[track_caller]
+    pub fn param(self, _name: &str, _value: impl Serialize) -> Self {
+        panic!("cannot set parameters on raw query - use regular query() for parameter binding");
+    }
```

Comment:
if this is a separate struct/impl, why we even need these methods that always panic?